kolibri/core/logger/test/test_api.py

```diff
@@ -582,56 +584,36 @@ def setUp(self):
         ]
         self.facility.add_admin(self.admin)
 
-    @patch(
-        "kolibri.core.logger.management.commands.exportlogs.AsyncListSerializer.data"
-    )
-    @patch("kolibri.core.api.CSVModelViewSet.get_queryset")
-    @patch("kolibri.core.logger.management.commands.exportlogs.Command._create_file")
-    def test_csv_download(self, _create_file_mock, get_queryset_mock, data_mock):
-        self.client.login(
-            username=self.admin.username,
-            password=DUMMY_PASSWORD,
-            facility=self.facility,
-        )
+    def test_csv_download(self):
         expected_count = ContentSummaryLog.objects.count()
-        data_mock.__iter__.return_value = ContentSummaryLog.objects.all()
-        get_queryset_mock.return_value = ContentSummaryLog.objects
+        _, filepath = tempfile.mkstemp(suffix=".csv")
```

Comment:
I remember I had to change this to a mock because creating tmp files where causing problem when running tests in Travis. Even using the `tmp_path` fixture (http://doc.pytest.org/en/latest/tmpdir.html) it failed. Do you know why this is not happening now?

Reply:
Not sure - never had any issues with temp files in travis tests before.

Reply:
I hate having such a bad memory but I do remember I had to use mocking because there were cases where the tmp files could not be created (My first implementation was exactly like the one you did it here but tests didn't pass in Travis). Maybe it was due to some previous Travis setup and the problem now is gone. I shoud have taken notes of it.